### ISSUE TITLE
Error during postinstall script

### ISSUE THREAD

**Issue by Phlarfl** [beakerbrowser/beaker#1325]
Operation System: Windows 10
Beaker version: Compiling from latest source (currently 0.8.2)

Running `npm install` results in the following error:

```
npm ERR! Error while executing:
npm ERR! C:\Program Files\Git\mingw64\bin\git.EXE ls-remote -h -t git://github.com/beakerbrowser/beaker-virtual-fs.git
npm ERR!
npm ERR! fatal: unable to look up github.com (port 9418) (Either the application has not called WSAStartup, or WSAStartup failed. )
npm ERR!
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\User\AppData\Roaming\npm-cache\_logs\2019-01-21T05_07_05_364Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! beakerbrowser@ postinstall: `cd app && npm install`
npm ERR! Exit status 1
```

**Comment by paul90** [comment]
You don't mention which version of Git you are using, but there was a know issue with Git for windows 2.19.2 which appears from the issues over there should have been fixed in their [2.20 release](https://github.com/git-for-windows/git/releases).

**Comment by Septimus** [comment]
As paul90 said. In addition, I was able to find this with a Google search: https://github.com/google/marzipano/issues/199
